src/test/java/Enums/BrowserName.java

```diff
@@ -0,0 +1,6 @@
+package Enums;
```

Comment:
Package names are written in all lower case

Reply:
@fnfldx This has been fixed now as well.